File: C#/KnowItTest/Transportations/Train.cs
public class Train : Transportation
{
    public Train() : base(new Dictionary<DayOfWeek, int>
    {
        { DayOfWeek.Monday, 50 },
        { DayOfWeek.Tuesday, 50 },
        { DayOfWeek.Wednesday, 50 },
        { DayOfWeek.Thursday, 50 },
        { DayOfWeek.Friday, 50 },
        { DayOfWeek.Saturday, 35 },
        { DayOfWeek.Sunday, 35 }
    })
    {
        // Constructor body 
    }
}

File: Java/Train.java
import java.util.Map;
import java.util.HashMap;
import java.time.DayOfWeek;

public class Train extends Transportation {
    public Train() {
        super(new HashMap<DayOfWeek, Integer>() {{
            put(DayOfWeek.MONDAY, 50);
            put(DayOfWeek.TUESDAY, 50);
            put(DayOfWeek.WEDNESDAY, 50);
            put(DayOfWeek.THURSDAY, 50);
            put(DayOfWeek.FRIDAY, 50);
            put(DayOfWeek.SATURDAY, 35);
            put(DayOfWeek.SUNDAY, 35);
        }});
    }
}